geode-for-redis/src/test/java/org/apache/geode/redis/internal/data/delta/ReplaceByteArrayDoublePairsDeltaUnitTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.redis.internal.data.delta;
+
+import static org.apache.geode.redis.internal.data.delta.DeltaType.REPLACE_BYTE_ARRAY_DOUBLE_PAIRS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+
+import junitparams.Parameters;
+import junitparams.naming.TestCaseName;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.apache.geode.redis.internal.data.RedisData;
+import org.apache.geode.redis.internal.data.RedisSortedSet;
+import org.apache.geode.redis.internal.data.RedisString;
+import org.apache.geode.test.junit.runners.GeodeParamsRunner;
+
+@RunWith(GeodeParamsRunner.class)
+public class ReplaceByteArrayDoublePairsDeltaUnitTest extends AbstractRedisDeltaUnitTest {
+  private final byte[] deltaBytes = "delta".getBytes();
+
+  @Test
+  public void testReplaceByteArrayDoublePairsDelta() throws Exception {
+    DataInputStream dis = getDataInputStream();
+    RedisSortedSet redisSortedSet = makeRedisSortedSet();
+
+    redisSortedSet.fromDelta(dis);
+
+    assertThat(redisSortedSet.zcard()).isEqualTo(1);
+    assertThat(redisSortedSet.zrank(deltaBytes)).isEqualTo(0L);
```

Comment:
Similar to the test in `AddByteArrayDoublePairsDeltaUnitTest`, we can beef up the assertions here a little by also asserting that the score for the replaced member is correct by making the hard-coded score on line 70 a field of the class and adding:
```
    byte[] scoreBytes = redisSortedSet.zscore(deltaBytes);
    assertThat(Coder.bytesToDouble(scoreBytes)).isEqualTo(score);
```

Reply:
Done.